README.md

```diff
@@ -30,8 +57,8 @@ All modules must have an accompanying doc comment that describes the general fun
 of the public functions, structs, enums, or other central elements of the module can be included.
 
 ### Testing
-All public and private functions must be covered by unit tests. Full coverage is required, which means all 
-branches of computation within the function needs to be tested. 
+All public and private functions must be covered by unit tests. Full coverage is required, which means all
+branches of computation within the function needs to be tested.
 
 ### Crates
 To avoid confusion and unnecessary dependencies, a list of utility crates are included. Note that this only includes crates
```

Comment:
Not sure I understand this. Why does that avoid confusion and unnecessary dependencies?

Reply:
The intention was to list the libraries the system use for specific functionality so different developers did not accidentally add multiple libraries that provide the same functionality. However, if that part of the README needs to be updated it should probably be in a different PR.